crates/bevy_pbr/src/light.rs

```diff
@@ -48,10 +52,13 @@ impl LightRaw {
 
         let proj = perspective.get_projection_matrix() * global_transform.compute_matrix();
         let (x, y, z) = global_transform.translation.into();
+
+        let mut color: [f32; 4] = light.color.into();
+        color[3] = light.intensity;
```

Comment:
Shouldn't this be written as `let mut color: [f32; 4] = (light.color * light.intensity).into();`?

Reply:
I guess we could premultiply that, yes. Right now it was just passing the intensity in the otherwise useless 4th element of the vec4.

Reply:
Done.